src/keycloak/chart/values.yaml

```diff
@@ -8,7 +8,14 @@ image:
   pullPolicy: IfNotPresent
 
 # renovate: datasource=github-tags depName=defenseunicorns/uds-config versioning=semver
-configImage: ghcr.io/defenseunicorns/uds/identity-config:0.2.1
+# configImage: ghcr.io/defenseunicorns/uds/identity-config:0.3.0
+configImage: ghcr.io/mjnagel/uds-identity-config:0.3.0-dev
```

Comment:
👀

Reply:
😅 temp change while the other PR was in flight so that I could test e2e and easily send this branch to others for validation.